modules/manage/pages/recovery-mode.adoc

```diff
@@ -0,0 +1,201 @@
+= Recovery Mode
+:description: 
+
+Recovery mode allows you to start Redpanda and form a stable cluster when it fails due to a system crash, OOM errors, or other issues that make a normal startup infeasible. 
+
+In recovery mode, cluster configuration changes and other manual administrative actions are permitted so you can repair and restore the failed cluster. Topic reads and writes are disabled.
+
+== About recovery mode
```

Comment:
This section looks good content-wise but feels a bit disjointed. Should we add some wording tying these together? E.g. preventing election storm is the direct consequence of not loading partitions. And preventing kafka traffic is a different but also important aspect of recovery mode. Also disabling some internal processes (listed in another section) can be added here as yet another aspect.

Reply:
The second sentence in particular might be better placed near the end of this section after we finish with the background on 'why'